src/App.js

```diff
@@ -1,25 +1,120 @@
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="mt-48 px-8 py-5 border-t-2 border-t-gray-200 w-full flex flex-wrap gap-8">
+      <span className="text-neutral-gray text-sm">© 2024 Tomo</span>
+      <a href="/about" className="text-neutral-gray text-sm underline underline-offset-2">About</a>
     </div>
   );
 }
 
-export default App;
+const CLASSES = [
+  {
+    id: 0,
+    title: "COMP_SCI 211-0",
+    description: "Fundamentals of Computer Programming II",
+    members: 67,
+    url: "/classes/comp-sci-211-0"
+  },
+  {
+    id: 1,
+    title: "COMP_ENG 203-0",
+    description: "Introduction to Computer Engineering",
+    members: 48,
+    url: "/classes/comp-eng-203-1"
+  },
+  {
+    id: 2,
+    title: "MATH 228-1",
+    description: "Multivariable Differential Calculus for Engineering",
+    members: 61,
+    url: "/classes/math-228-1"
+  },
+  {
+    id: 3,
+    title: "GEN_ENG 206-1",
+    description: "Honors Engineering Analysis 1",
+    members: 56,
+    url: "/classes/gen-eng-206-1"
+  },
+  {
+    id: 4,
+    title: "CONDUCT 393-0",
+    description: "Orchestral Organizations",
+    members: 77,
+    url: "/classes/conduct-393-0"
+  }
+];
+
+function ClassCard(props) {
+  return (
+    <article className="group relative transition hover:-translate-y-1 h-full cursor-pointer">
+      <a href={props.url}>
+        <div className="border-2 border-gray-200 rounded-2xl p-8 h-full group-hover:shadow-md">
+          <h2 className="text-3xl text-neutral-black font-bold font-display">{props.title}</h2>
+          <p className="text-md text-neutral-gray">{props.description}</p>
+          <p className="mt-1 text-md text-neutral-gray">{props.members} members</p>
+        </div>
+      </a>
+      <button className="text-neutral-black hover:text-red-500 absolute top-4 right-4"
+              aria-label={`Remove ${props.title}`}
+              onClick={props.handleRemove}>
+        <FontAwesomeIcon icon={faTrashCan} size="lg"/>
+      </button>
+    </article>
+  );
+}
+
+function App() {
+  const [classes, setClasses] = useState(CLASSES);
+
+  const handleRemove = (index) => {
+    const newClasses = [...classes];
+    newClasses.splice(index, 1);
+    setClasses(newClasses);
+  };
+
+  useEffect(() => {
+    console.log(classes);
+  }, [classes]);
+
+  return (
+    <>
+      <NavBar/>
+      <div className="px-8 py-10">
+        <h1 className="text-5xl text-neutral-black font-extrabold font-display">Classes</h1>
+        <div className="mt-8 grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-6 items-stretch">
+          {classes.map((c, idx) => (
```

Comment:
optional, but you could add a placeholder here for if there are no classes present (e.g. if all classes were removed). would look something like this

{classes.length === 0 ? (
  <p className="text-neutral-gray text-lg">No classes available.</p>
) : (
  <div className="mt-8 grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-6 items-stretch">
    {classes.map((c) => (
      <ClassCard {...c} handleRemove={() => handleRemove(c.id)} key={c.id}/>
    ))}
  </div>
)}